contracts/deprecated/RelayerIncentivize.sol

```diff
@@ -55,61 +55,17 @@ contract RelayerIncentivize is IRelayerIncentivize, System, IParamSubscriber {
         alreadyInit = true;
     }
 
-    receive() external payable { }
+    receive() external payable {
+        revert("deprecated");
+    }
 
     function addReward(
         address payable headerRelayerAddr,
         address payable packageRelayer,
         uint256 amount,
```

Comment:
`calculateTransferRelayerWeight` and `calculateHeaderRelayerWeight` no longer needed

Reply:
it is `external pure`, the deletion rule is not met

Reply:
mark deprecated then

Reply:
removed